src/pages/Index.vue

```diff
@@ -0,0 +1,252 @@
+          {{feature.title}}
+        </div>
+        <div class="col-1 q-pt-sm">
+          {{feature.subtitle}}
+        </div>
+      </div>
+      <q-space />
+    </div>
+
+    <div id="download" class="row q-py-xl">
+      <q-space />
+        <div class="text-center">
+            <div class="text-h3 text-grey-8 q-pb-xl">
+              Optimize your Work<strong class="text-secondary">Flow</strong>!
+            </div>
+          <q-btn
+            type="a"
+            href="https://github.com/Flow-Launcher/Flow.Launcher/releases"
+            icon="mdi-github"
+            size="lg"
+            color="secondary">
+            &nbsp;Download from GitHub
+          </q-btn>
+          <p class="text-caption">
+            <strong>Flow</strong> is only available for Windows™ 7 and Newer
+          </p>
+        </div>
+      <q-space />
+    </div>
+  </q-page>
+</template>
+
+<script>
+export default {
+  name: 'PageIndex',
+  data() {
+    return {
+      query: 'FlowLauncher',
+      possible_results: [
+        {
+          icon: '/images/logo.svg',
+          title: 'FlowLauncher - a launcher for Windows',
+          subtitle: 'Boost your productivity!',
+          link: '',
+          color: '',
+        },
+        {
+          icon: '/images/logo.svg',
+          title: 'FlowLauncher - Features',
+          subtitle: 'Explore the possibilities!',
+          link: '',
+          color: '',
+        },
+        {
+          icon: '/images/logo.svg',
+          title: 'FlowLauncher - Download',
+          subtitle: 'Download from GitHub!',
+          link: 'https://github.com/Flow-Launcher/Flow.Launcher/releases',
+          color: '',
+        },
+        {
+          icon: '/images/logo.svg',
+          title: 'FlowLauncher - Trouble?',
+          subtitle: 'Report an Issue!',
+          link: 'https://github.com/Flow-Launcher/Flow.Launcher/issues',
+          color: '',
+        },
+      ],
+      features: [
+        {
+          icon: 'mdi-magnify', title: 'Search Everything', subtitle: 'Application, Files, Bookmarks, Youtube and more!',
+        },
+        {
+          icon: 'mdi-toy-brick', title: 'Plugin Suppport', subtitle: 'Extend your Flow experience with a variety of Plugins!',
+        },
+        {
+          icon: 'mdi-console-line', title: 'Native Shell Support', subtitle: 'Execute (Power)Shell commands directly with FlowLauncher!',
+        },
+      ],
+    };
+  },
+  computed: {
+    results() {
+      const resultList = [];
+
+      if (this.query.trim() === '') {
+        return resultList;
+      }
+
+      const mathRegex = /^(?:\d+|\+|-|\/|\*|\(|\))*$/gm;
+      if (this.query.match(mathRegex)) {
+        const mathString = this.query.replace(/[^-()\d/*+.]/g, '');
+
+        resultList.push(
+          {
+            // eslint-disable-next-line no-eval
+            icon: '/images/calculator.png', title: eval(mathString), subtitle: 'Calculated from your expression', link: '', color: '',
```

Comment:
The calculator function may need to be deleted. I can't figure out another better way to handle the `eval` function.

Just like the error says: `eval can be harmful  no-eval`

Reply:
Yeah, eval *can* be harmful, but In this context and with the string sanitizing in front of it it's harmless. Especially in our case - the website never pushes any data to a database, so the only experience someone malicious could break is his own one.